install-nvm.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+wget https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh
+php -r '
+    $hash = "dd4b116a7452fc3bb8c0e410ceac27e19b0ba0f900fe2f91818a95c12e92130fdfb8170fec170b9fb006d316f6386f2b";
+    if (hash_file("sha384", "install.sh") === $hash) {
+    echo "Installer verified";
+    } else {
+    echo "Installer corrupt";
+    unlink('install.sh');
```

Comment:
Should also add a 'cleanup' thing to remove __install-nvm.sh e.g. https://github.com/silverstripe/gha-trigger-ci/blob/1/action.yml#L95

Reply:
Done